app/views/core_induction_programmes/mentor_material_parts/show.html.erb

```diff
@@ -3,7 +3,12 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <% if policy(@mentor_material_part.mentor_material).edit? %>
+      <div class="govuk-button-group">
         <%= govuk_link_to "Edit mentor material", edit_mentor_material_path(@mentor_material_part.mentor_material), button: true %>
+        <%= govuk_link_to "Edit mentor material part", edit_mentor_material_part_path(@mentor_material_part), button: true %>
+        <%= govuk_link_to "Split mentor material part", mentor_material_part_split_path(mentor_material_part_id: @mentor_material_part.id), button: true %>
+        <%= govuk_link_to "Delete mentor material part", mentor_material_part_show_delete_path(mentor_material_part_id: @mentor_material_part.id), button: true, class: "govuk-button--secondary" %>
+      </div>
```

Comment:
Can we add navigation to the bottom of the page (to next/previous, like lesson parts have?)